packages/swing-store/src/snapStore.js

```diff
@@ -53,18 +65,14 @@ const noPath = /** @type {import('fs').PathLike} */ (
   /** @type {unknown} */ (undefined)
 );
 
-const sink = () => {};
-
 /**
+ * @param {*} db
```

Comment:
Oh, I just noticed that the `root` parameter is obsolete, now that the snapshots are included in the directory. That also means `swingStore.js` no longer needs to condition the creation of a `snapStore` on the on-disk-ness of the swingstore: it can always build a snapstore.

Reply:
I thought it needed that for the temp file (which is an actual file) that was being passed along to XS, but I guess it uses the `tmp` package to do that.   So, yeah; I'll simplify.

Reply:
oh also `ephemeralSnapStore` goes away

Reply:
Weird that eslint didn't complain about the unused parameter.

Reply:
Getting rid of the `root` parameter eliminated a bunch of code in tests all over the place.  Do you want to re-review or are we good to go?

Reply:
Ugh this change is really pernicious because going from a `string` to a `*` does not cause any type checks to fail.